shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -140,6 +144,18 @@
 
   settings.dart_library_sources_kernel =
       make_mapping_callback(kPlatformStrongDill, kPlatformStrongDillSize);
+
+  // Tell Dart in JIT mode to not use integer division on armv7
+  // Ideally, this would be detected at runtime by Dart.
+  // TODO(dnfield): Remove this code  https://github.com/dart-lang/sdk/issues/24743
+  size_t size;
+  cpu_type_t type;
+
+  size = sizeof(type);
+  sysctlbyname("hw.cputype", &type, &size, NULL, 0);
+  if (type == CPU_TYPE_ARM) {
```

Comment:
Does this have to be a runtime check?
Wouldn't it be possible to handle this similarly to how `kDartWriteProtectCodeArgs` is set up: https://github.com/flutter/engine/blob/master/runtime/dart_vm.cc#L319

Reply:
We could do this as a compile define, but I couldn't quite find the one for ARM vs ARM64. I don't think the one you linked is it (but the overall method there could work).

Reply:
I can add the define - not sure if I'm doing it in the ideal way but will push that up.

Reply:
Changed this to a feature define